manifests/e/EdrawSoft/EdrawMax/CN/12.0.8.978/EdrawSoft.EdrawMax.CN.locale.en-US.yaml

```diff
@@ -0,0 +1,55 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-10
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: EdrawSoft.EdrawMax.CN
+PackageVersion: 12.0.8.978
+PackageLocale: en-US
+Publisher: EdrawSoft Co.,Ltd.
+PublisherUrl: https://www.edrawsoft.cn/
+PublisherSupportUrl: https://www.edrawmax.cn/helpcenter/pc/home
+PrivacyUrl: https://www.edrawsoft.cn/private-policy/
+Author: SHENZHEN EDRAW SOFTWARE CO.,LTD.
+PackageName: EdrawMax
+PackageUrl: https://www.edrawsoft.cn/edrawmax/
+License: Proprietary
+LicenseUrl: https://www.edrawsoft.cn/edrawmax-term-services/
+Copyright: Copyright © 2023 EdrawSoft. All rights reserved.
+# CopyrightUrl:
+ShortDescription: One-stop office diagram tool
+Description: EdrawMax is a design software that integrates office diagramming, engineering diagramming, image-text editing and flyer design, with which you can create all kinds of diagrams easily without any drawing skills.
+# Moniker:
+Tags:
+- chart
+- design
+- diagram
+- draw
+- gantt
+- graph
+- graphics
+- mind-map
+- mind-mapping
+- mindmap
+- uml
+# Agreements:
+ReleaseNotes: |-
+  【新增】AI智能助手
+  AI智能助手是风靡全球的聊天机器人，它能够帮助扩展您的思维，根据您输入的问题来进行文字创作，创作后可将文字生成绘图插入到画布中
+  • 方式一：输入一句话，AI为您生成创意绘图
+  • 输入指令→执行指令→完成作图(思维导图、列表、文本框)
+  • 方式二：选中一句话，AI为您生成脑暴内容
+  • 选中形状→右键快捷菜单→AI智能生成内容
+
+  【新增】OCR图片文字提取
+  上传一张图片，通过OCR识别可以提取出图片中的文字，并直接将文字应用到画布中
```

Comment:
```suggestion
[New] AI smart assistant
   AI smart assistant is a chat robot that is popular all over the world. It can help expand your thinking and create text according to the questions you input. After creation, the text can be generated and drawn and inserted into the canvas.
   • Method 1: Enter a sentence, AI will generate a creative drawing for you.
   • Enter instructions → execute instructions → complete the drawing (mind map, list, text box.)
   • Method 2: Select a sentence, AI will generate brainstorm content for you.
   • Select the shape → right-click shortcut menu → AI intelligently generate content.
   [New] OCR image text extraction.
   Upload a picture, the text in the picture can be extracted through OCR recognition, and the text can be directly applied to the canvas.
```